### PR TITLE
[JENKINS][FEAT] Add cronjob to cleanup preview

### PR DESCRIPTION
**Pull request by sidohaakma**

### CODE REVIEW

File: charts/molgenis-jenkins/resources/cleanup/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        stage("Cleanup") {
+            steps {
+                container('vault') {
+                    sh "mkdir /home/jenkins/.rancher"
+                    sh(script: 'vault read -field=value secret/ops/jenkins/rancher/cli2.json > /home/jenkins/.rancher/cli2.json')
+                }
+                container('rancher') {
+                    sh "rancher context switch dev-molgenis"
+                    sh "rancher app | awk '{print $2}' | grep preview- > preview.list"
+                    sh "xargs rancher app delete < preview.list"
```
Comment (fdlk) [inline charts/molgenis-jenkins/resources/cleanup/Jenkinsfile]: but but but... My previews!

Comment (fdlk) [reply charts/molgenis-jenkins/resources/cleanup/Jenkinsfile]: More constructively: This is a bit rough I think. If we figure out how to put a bit of age logic in the loop somewhere we can prevent the job from cleaning up someone's carefully prepared demo-for-tomorrow.

Comment (fdlk) [reply charts/molgenis-jenkins/resources/cleanup/Jenkinsfile]: Alternatively, we can arrange a way for devs to deploy non-preview server that won't match this job's filter.

**Comment by fdlk** [comment]
I ran into a talk where folks use https://github.com/deislabs/osiris to scale their previews to zero when they're not in use

File: charts/molgenis-jenkins/resources/cleanup/Jenkinsfile (hunk 1 above)
Comment (sidohaakma) [reply charts/molgenis-jenkins/resources/cleanup/Jenkinsfile]: I would prefer that. There is already a story to amend "mark" with this functionality: http://wiki.gcc.rug.nl/ticket/6841.

File: charts/molgenis-jenkins/values.yaml [hunk 2]
```diff
@@ -110,6 +110,71 @@ jenkins:
...
+              <enableNotifications>true</enableNotifications>
+            </org.thoughtslive.jenkins.plugins.hubot.config.HubotJobProperty>
+            <hudson.model.ParametersDefinitionProperty>
+              <parameterDefinitions>
+                <hudson.model.ChoiceParameterDefinition>
+                  <name>ACTION</name>
+                  <description>What action to perform</description>
+                  <choices class="java.util.Arrays$ArrayList">
+                    <a class="string-array">
+                      <string>install</string>
```
Comment (fdlk) [inline charts/molgenis-jenkins/values.yaml]: Options got copy-pasted?

File: charts/molgenis-jenkins/resources/cleanup/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
+pipeline {
+    agent {
+        kubernetes {
+            label 'shared'
+        }
+    }
+    triggers {
+        cron('H 1 * * *')
```
Comment (fdlk) [inline charts/molgenis-jenkins/resources/cleanup/Jenkinsfile]: Move the trigger to the job definition so we can't forget to trigger it after an upgrade.

**Pull request merged**